FlowersStore/Scenes/FlowersList/FlowersListPresenter.swift

```diff
@@ -20,16 +20,26 @@ final class FlowersListPresenter {
     private weak var output: FlowersListPresenterOutput?
     private let interactor: FlowersListInteractorInput
     
+    private var teste: String?
+    
     init(interactor: FlowersListInteractorInput = FlowersListInteractor()) {
         self.interactor = interactor
     }
     
     func setOutput(_ output: FlowersListPresenterOutput?) {
         self.output = output
     }
+    
+    func setName() {
+        
+    }
 }
 
 extension FlowersListPresenter: FlowersListPresenterInput {
+    func koe() {
```

Comment:
<!--
  0 failure: 
  1 warning:  Function 'koe()' ...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Scenes/FlowersList/FlowersListPresenter.swift;
  Line: 39;
-->

- :warning: Function 'koe()' is unused